std_msgs/Header header
int64 thrust_offset    # thrust_offset reading


